src/Appwrite/Utopia/Database/Validator/Queries/Indexes.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Appwrite\Utopia\Database\Validator\Queries;
+
+class Indexes extends Base
+{
+    public const ALLOWED_ATTRIBUTES = [
+        'key',
+        'type',
+        'status',
```

Comment:
Let's add `error` here too

Reply:
didn't add it, there is no error property in db schema for indexes